ebpf/event_common.go

```diff
@@ -58,6 +60,9 @@ type ConnectionStats struct {
 	RecvBytes uint64 `json:"recv_bytes"`
 
 	Retransmits uint32 `json:"retransmits"`
+
+	// collected is used internally to know if this connection stats has been collected or not
```

Comment:
I think this could use some additional clarity on the "why" of collected instead of just the "what". Why do we need this field, what purpose does it serve. At a glance it's unclear when this is used and `collected` is a pretty generic name.